### PR TITLE
Fix:Basic UI Fixes

### PR DESCRIPTION
**Pull request by rishuriya**
First of all sorry for delay, As i mention in one of my comment i was bit busy with my end sem exam, and one of project for Amrita hospital Fbd, now it ended so i can fully focus here, This PR has basic UI changes

Before           |  Now
:-------------------------:|:-------------------------:
Screenshot_20230228-024507_ListenBrainz_Debug | Screenshot_20230228-024531_ListenBrainz_Debug
The Like button is black which is not visible in dark mode| Changed it to Red as like represent red so it has the red color and also visible in dark mode

Before           |  Now
:-------------------------:|:-------------------------:
Screenshot_20230228-023617_ListenBrainz_Debug|Screenshot_20230228-023539_ListenBrainz_Debug
As you clearly see the change, First song is more toward Left| Now its on the position where other songs are.

These are the basic UI changes, @akshaaatt if you found any other UI changes which can be done i can fix. 

and apart from that on profile page, i started working on figma, i will share the design soon.

### CODE REVIEW

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt [hunk 1]
```diff
@@ -249,7 +249,7 @@ fun PlayerScreen(
                                 }
                             }
                         },
-                    tint = if (!listenLiked) Color.Red else Color.Black
+                    tint = Color.Red
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt:252]: We shouldn't be doing this. Rather that this, change the else condition to pick the onSurface color I think

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/songs/SongScreen.kt [hunk 2]
```diff
@@ -220,7 +220,9 @@ fun SongScreen() {
                     Column(horizontalAlignment = Alignment.CenterHorizontally) {
                         Box(
                             modifier = Modifier
-                                    .padding(10.dp)
+                                    .padding(
+                                        start=24.dp,
+                                        end = 24.dp,)
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/songs/SongScreen.kt:225]: Please clean this to adopt to the lint

**Review by akshaaatt — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt (hunk 1 above)
Comment (rishuriya) [reply app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt]: yeah, i think it`s a good idea, thanks for suggesting

**Comment by rishuriya** [comment]
@akshaaatt did the changes as suggested.

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt [hunk 3]
```diff
@@ -249,7 +249,7 @@ fun PlayerScreen(
                                 }
                             }
                         },
-                    tint = if (!listenLiked) Color.Red else Color.Black
+                    tint = MaterialTheme.colorScheme.onSurface
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt:252]: I think the if else conditions should be retained here and the color should be red when liked and in the else condition we can have the onsurface color

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/artist/ArtistScreen.kt [hunk 4]
```diff
@@ -74,6 +77,7 @@ private fun ArtistsScreen(
                 }
             ) {
                 Column(horizontalAlignment = Alignment.CenterHorizontally) {
+//                    Spacer(modifier = Modifier.width(100.dp)) // added Spacer
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/artist/ArtistScreen.kt:80]: What's the logic behind commenting this and adding it for review?

**Review by akshaaatt — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/artist/ArtistScreen.kt (hunk 4 above)
Comment (rishuriya) [reply app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/artist/ArtistScreen.kt:80]: oh that i forgot to remove it. i am trying to add space as in artist the left box is nearly touching the edge. i will remove it

**Review by akshaaatt — CHANGES_REQUESTED** [review]
LGTM! Can you please rebase/update your branch to the latest changes on main to avoid conflicts?

**Comment by rishuriya** [comment]
yeah sure

**Pull request merged**